models/users.js

```diff
@@ -863,6 +868,68 @@ const getNonNickNameSyncedUsers = async () => {
     throw err;
   }
 };
+const addGithubUserId = async () => {
+  try {
+    const usersNotFound = [];
+    let countUserFound = 0;
+    let countUserNotFound = 0;
+
+    const requestOptions = {
+      method: "GET",
+      headers: {
+        "Content-Type": "application/json",
+        Authorization:
+          "Basic " + btoa(`${config.get("githubOauth.clientId")}:${config.get("githubOauth.clientSecret")}`),
+      },
+    };
+    const usersSnapshot = await firestore.collection("users").get();
+    const totalUsers = usersSnapshot.docs.length;
+    const batchCount = Math.ceil(totalUsers / 500);
+    // Create batch write operations for each batch of documents
+    for (let i = 0; i < batchCount; i++) {
+      const batchWrite = firestore.batch();
+      const batchWrites = [];
+      for (const userDoc of usersSnapshot.docs.slice(i * 500, (i + 1) * 500)) {
```

Comment:
```suggestion
      for (let j = i * 500 ; j < (i + 1) * 500; j++) {
      const userDoc = usersSnapshot.docs[j];
```
or just remove this inner loop

Reply:
Fixed